Kubeedge/proposals/Node-Group-Management.md

```diff
@@ -82,11 +117,31 @@ After `kube-scheduler` running all built-in filter plugins, it will send the fil
+`GroupManagementControllerManager` contains three controllers, called `NodeGroupController`, `PropagationPolicyController` and `OverridePolicyController`.
+
+#### NodeGroupController
+`NodeGroupController` is responsible for reconciling the NodeGroup Object which will collect nodes belonging to the NodeGroup according to node names and labels and fill the status field. `NodeGroupController` only watches the nodegroup API.
+
+#### PropagationPolicyController
+`PropagationPolicyController` is responsible for reconciling the current replica number of pods in each nodegroup with the desired distribution status which is defined in the PropagationPolicy. It will evict pods in nodegroups where they should not run and also evict pods in nodegroups when they are redundant(which means the current replica number of pods in the nodegroup exceeds what the policy desires). 
+
+`PropagationPolicyController` should watch configmap `groupmanagement-config`. It will dynamically add/delete informers according to the GVK set in the config. These informers will react to the modification of labels. If it find `groupmanagement.kubeedge.io/required-propagationpolicy`, it will check if the propagation policy actually exists and add binding label for it. Here, we leave extension space before adding binding label. It will also watch PropogationPolicy and NodeGroup resources and do the reconcilation.
 
-`NodeGroupController` is responsible for reconciling the NodeGroup Object which will collect nodes belonging to the NodeGroup according to node names and labels and fill the status field.
+#### OverridePolicyController
+`OverridePolicyController` is responsible for reconciling the current pod instance edition in each nodegroup with desired pod instance edition which is defined in the OverridePolicy. If inconsistent, it will modify relative paths of pods at APIServer, and then pods will automatically restart. 
 
-`PropagationPolicyController` is responsible for reconciling the current replica number of pods in each nodegroup with the desired distribution status which is defined in the PropagationPolicy. It will evict pods in nodegroups where they should not run and also evict pods in nodegroups when they are redundant(which means the current replica number of pods in the nodegroup exceeds what the policy desires).   
+`OverridePolicyController` should watch configmap `groupmanagement-config`. It will dynamically add/delete informers according to the GVK set in the config. These informers will react to the modification of labels. If it find `groupmanagement.kubeedge.io/required-overridepolicy`, it will check if the override policy actually exists and add binding label for it. Here, we leave extension space before adding binding label. Currently, we can apply the required override policy for this pod. It will also watch OverridePolicy and NodeGroup resources and do the reconcilation.
```

Comment:
`It will also watch OverridePolicy and NodeGroup resources and do the reconcilation.`

reconcilation --> reconciliation

Reply:
fixed